src/format.rs

```diff
@@ -285,6 +332,11 @@ impl CodecConfig {
         c
     }
 
+    /// Returns a reference to the vector of payload configurations.
+    pub fn get_configs(&self) -> &Vec<PayloadParams> {
```

Comment:
Let's rename the internal field and this function `params()`. That's the name in the rest of the API.

Reply:
will fix